deploy/kubernetes/deploy-kiali-to-kubernetes.sh

```diff
@@ -69,7 +91,8 @@ fi
 # If asking for the last release (which is the default), then pick up the latest release.
 # Note that you could ask for "latest" - that would pick up the current image built from master.
 if [ "${IMAGE_VERSION}" == "lastrelease" ]; then
-  version_we_want=$(curl https://api.github.com/repos/kiali/kiali/releases/latest 2> /dev/null |\
+  get_downloader
+  version_we_want=$(${downloader} https://api.github.com/repos/kiali/kiali/releases/latest 2> /dev/null |\
```

Comment:
can this be named a bit more clearly? perhaps kiali_version ?

Reply:
@abonas done - "$kiali_version_we_want"

Reply:
just to be clear -- I left the "_we_want" portion to make it more obvious to the person reading the code that this is going to be the kiali version that the user wants to install based on what the github API returns to us - it isn't the version we have downloaded/installed yet. Note that the curl (downloader) here is only grabbing the version string of the latest github release from the github API - we are saying "the github API is returning the version of kiali that we want to use." Later on in the script we will download this version that we want.